src/main/java/bio/terra/cli/apps/DockerAppsRunner.java

```diff
@@ -51,6 +52,9 @@ public DockerAppsRunner(GlobalContext globalContext, WorkspaceContext workspaceC
     this.dockerClient = null;
   }
 
+  // ====================================================
+  // Docker images
```

Comment:
not this PR, but when it starts getting useful to provide sub-class structure, it can be a sign that it's worth breaking this out into helper classes and utilities.

Reply:
Agreed. I think breaking out the Docker images and container sections into a helper class could be a way to do this. Will keep in mind for future clean-up work.